src/tools/kata-ctl/src/main.rs

```diff
@@ -22,11 +23,32 @@ use ops::check_ops::{
 use ops::env_ops::handle_env;
 use ops::exec_ops::handle_exec;
 use ops::volume_ops::handle_direct_volume;
+use slog::{error, o};
+
+macro_rules! sl {
+    () => {
+        slog_scope::logger().new(o!("subsystem" => "main"))
+    };
+}
 
 fn real_main() -> Result<()> {
     let args = KataCtlCli::parse();
 
-    match args.command {
+    let log_level = match args.log_level {
+        None => slog::Level::Info,
+        Some(level_arg) => level_arg.get_slog_level(),
+    };
+
+    let (logger, _guard);
+    if args.json_logging {
```

Comment:
Do not write this kind of initialization.

Use
```Rust
let (logger, _guard) = if args.json_logging {
...
} else {
...
}
```
instead.

Reply:
Done, thanks